src/client/src/ui/analytics/components/LastPosition.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import numeral from 'numeral'
+import { LastPositionWrapper, LastPositionStyle, USDspan } from './styled'
+
+interface LastPositionProps {
+  lastPos?: number | undefined
+}
+
+interface FormattedLastPosition {
+  lastPosition: string | undefined
+  color: string
+}
+
+const formatLastPosition: (lastPos: number) => FormattedLastPosition = (lastPos: number) => {
+  let lastPosition = numeral(lastPos).format()
+  let color = ''
+  if (lastPos > 0) {
+    color = 'green'
```

Comment:
Also it would be better to use string enums to represent the colours eg.
```
enum IndicatorColors = {
  RED = 'red',
  GREEN = 'green,
}
IndicatorColors.RED etc
```